src/services/items/fluent-schema.ts

```diff
@@ -18,6 +18,7 @@ const settings = S.object()
   .prop('isPinned', S.boolean())
   .prop('tags', S.array())
   .prop('showChatbox', S.boolean())
+  .prop('isResizable', S.boolean())
```

Comment:
Now that I think about it, does it make sense that this is set in settings that can be applied to all items? Do we have a way to restrict the type to apps only?

Reply:
In the builder, the setting will not be shown when the item type is not an app. In the player, the setting is only used when the item type is an app. But yes you could send a request to add this setting to any item type. I did not see a mechanism to prevent some settings based on the item type.